packages/code-snippet/src/CodeSnippetWidget.tsx

```diff
@@ -213,19 +213,21 @@ class CodeSnippetDisplay extends React.Component<ICodeSnippetDisplayProps> {
         onClick: (): void => {
           CodeSnippetService.deleteCodeSnippet(codeSnippet).then(
             (response: any): void => {
```

Comment:
might be worth renaming this arg `deleted: boolean` (or something like that) for clarity